### PR TITLE
added test to remove vowels of string

### PR DESCRIPTION
**Pull request by sureshrp**

### CODE REVIEW

File: alg-excer3.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/*Create a function that takes a string and returns a new string with all vowels removed.
+Examples
+removeVowels("I have never seen a thin person drinking Diet Coke.")
+➞ " hv nvr sn  thn prsn drnkng Dt Ck."*/
+
+const removingVowel = str => {
+	var newString = "";
```
Comment (koding-101) [inline alg-excer3.js]: nit tabbing otherwise looks good!

File: alg-excer3.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    }
+  }
+  return newString;
+}
+
+function testRemovingVowel(str) {
+  console.log(removingVowel(str));
+}
+
+testRemovingVowel("I have never seen a thin person drinking Diet Coke.");
```
Comment (koding-101) [inline alg-excer3.js]: you should include what it suppose to be. not just a console.log

File: alg-excer3.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+testRemovingVowel("I");
+testRemovingVowel("hello");
+testRemovingVowel("");
+testRemovingVowel("My age is 0, 0 si ega ym");
+testRemovingVowel("  WOW! Ren is so active?");
```
Comment (koding-101) [inline alg-excer3.js]: you should think of more edge cases.

"space space"
"AEIOU"
" AEIOU "
"aeiou"
"  aeiou "

File: alg-excer3.js (hunk 1 above)
Comment (sureshrp) [reply alg-excer3.js]: not showing in my VSC, already confirmed

File: alg-excer3.js (hunk 2 above)
Comment (sureshrp) [reply alg-excer3.js]: updated

File: alg-excer3.js (hunk 3 above)
Comment (sureshrp) [reply alg-excer3.js]: added

**Pull request merged**